### PR TITLE
[Tommy Collison] DB Zoos

### PR DESCRIPTION
**Pull request by tommycollison**
@johnoro 🦁 🐯 🐅 🐒

### CODE REVIEW

**Comment by tommycollison** [comment]
- [ x ] POST /api/zoos
- [ x ] GET /api/zoos
- [ x ] GET /api/zoos/:id
- [ x ] DELETE /api/zoos/:id
- [ x ] PUT /api/zoos/:id

File: README.md [hunk 1]
```diff
@@ -7,6 +7,7 @@ Topics:
 - Knex
 - Create/Read/Update/Delete operations
 
+## Description
```
Comment (johnoro) [inline README.md]: That's two descriptive descriptions.

File: index.js [hunk 2]
```diff
@@ -1,14 +1,87 @@
...
 
 // endpoints here
 
-const port = 3300;
+server.post('/api/zoos', (req, res) => {
+  const animal = req.body;
+  console.log(`hi from line 18`)
+  db('zoos').insert(animal)
+  .then(ids => { 
+      console.log(`hi from line 21`)
```
Comment (johnoro) [inline index.js]: Clean out your old `console.log`s that were just for testing!

File: index.js [hunk 3]
```diff
@@ -1,14 +1,87 @@
...
+  .catch(err => {
+    res.status(500).json({err: 'Failed to insert animal'});
+  });
+});  
+
+
+server.get('/api/zoos', (req, res) => {
+  db('zoos')
+  .then(rows => {
+      res.json(rows)
```
Comment (johnoro) [inline index.js]: Explicitly send a `200` status! (`res.status(200).json(rows)`)

**Pull request closed without merging**